ott/geometry/costs.py

```diff
@@ -32,9 +36,11 @@ class CostFn(abc.ABC):
 
   Cost functions evaluate a function on a pair of inputs. For convenience,
   that function is split into two norms -- evaluated on each input separately --
-  followed by a pairwise cost that involves both inputs, as in
+  followed by a pairwise cost that involves both inputs, as in:
+
+  .. math::
 
-  c(x,y) = norm(x) + norm(y) + pairwise(x,y)
+    c(x,y) = norm(x) + norm(y) + pairwise(x,y)
```

Comment:
I think this won't look good with maths

Reply:
Think it renders ok-ish, maybe we can just replace `norm(x)` with `||x||`